assignments/mgmt-groups/mg-dts002/assign.appgateway_diagnostics.json

```diff
@@ -0,0 +1,30 @@
+{
+    "properties": {
+        "metadata": {
+            "assignedBy": "github.com/hmcts/azure-policy"
+        },
+        "description": "Forwards all App Gateway Logs to an Azure Event Hub",
+        "displayName": "HMCTS Collect App Gateway Logs",
```

Comment:
I would add the subscription name here as well just to make it easier to see at a glance

Reply:
not been added to other assignments so will add to them after, makes sense

Reply:
Yeah, I had only thought to do it when working on the keyvault one and figured it would be best to make it standard